xbmc/GUIInfoManager.cpp

```diff
@@ -7182,7 +7187,7 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
     case PLAYER_PLAYING:
       {
         float speed = g_application.m_pPlayer->GetPlaySpeed();
-        bReturn = (speed >= 0.75 && speed <= 1.55);
+        bReturn = (speed == 1.0);
```

Comment:
In general, windows warns on conversion of double to float:

`warning C4244: conversion from 'double' to 'float', possible loss of data`

Kodi currently has 200 of such warnings.

I don't think that this line generates a warning on Windows, but I prefer to always match precisions because it makes it easier to reason about the code I maintain.